client/extensions/woocommerce/app/settings/shipping/save-button.js

```diff
@@ -61,11 +79,11 @@ class ShippingSettingsSaveButton extends Component {
 
 		if ( finishedInitialSetup ) {
 			return wcsEnabled
-				? <Button onClick={ this.save } primary>{ translate( 'Save' ) }</Button>
+				? <Button onClick={ this.save } primary busy={ isSaving } disabled={ isSaving }>{ translate( 'Save' ) }</Button>
```

Comment:
@kellychoffman In this page there will be 3 sections: `Label settings`, `Shipping Zones` and `Package Manager`. The `Shipping Zones` section can't be edited directly (all the changes happen in separate pages, each with its own `Save` button). Does it make sense to have a global `Save` button in the `Shipping` page, which will only affect 2 of the 3 sections in the page? I think it may be confusing, and we should consider for each section to have its own `Save` button.